Source/AwsCommonRuntimeKit/http/HTTPRequest.swift

```diff
@@ -4,7 +4,7 @@ import AwsCHttp
 import AwsCIo
 import AwsCCommon
 
-public class HTTPRequest: HTTPMessage {
+public class HTTPRequest: HTTPRequestBase {
```

Comment:
I have not renamed this class to avoid breaking the API.

Reply:
/// docs